### ISSUE TITLE
Please add option for smallfiles

### ISSUE THREAD

**Issue by ffxsam** [arunoda/meteor-up-legacy#792]
The journal file really gets out of control, it would be great if mupx had an option to set `smallfiles` to `true` to avoid smaller disks running out of space.

**Comment by MasterJames** [comment]
I think one can hack those settings in...
https://github.com/arunoda/meteor-up/blob/mupx/templates/linux/mongodb.conf
I think I left that or similar set in my now aging Pull Request for mupx.
There are various ways to change that. Other then a pull request of your own. One is editing the file where npm installs it, or on the server after.
I'll finish by saying +1 to that and the 30+ potential improvements lost in PR limbo.

**Comment by ffxsam** [comment]
Right, you can modify the mongodb.conf file but that gets overwritten every time you deploy. I'll plan on doing a PR to add this option to mup.json.

**Comment by MasterJames** [comment]
Yes thanks.

**Comment by arunoda** [comment]
@MasterJames we are moving mup into Kadira these days. See: https://github.com/kadirahq/meteor-up

Idea of this to speed up the mup development and get community contributions a lot faster. This will have essential tests and so on. We can port existing mupx features in two weeks. Then we can add features faster.

I'll do a blog post next week or after that.
Thank you all.

**Comment by ffxsam** [comment]
I'll be sure to do the PR there. Thanks Arunoda!

**Comment by MasterJames** [comment]
Oh good to hear thanks for noting that.
As a final note changing the npm installed version will survive re-deployments unless npm install mupx is part of a dockerfile or build script etc.
